transformers/events/vat_flux/converter_test.go

```diff
@@ -19,23 +19,41 @@ package vat_flux_test
 import (
 	"github.com/ethereum/go-ethereum/common"
 	"github.com/ethereum/go-ethereum/core/types"
+	"github.com/makerdao/vdb-mcd-transformers/test_config"
+	"github.com/makerdao/vdb-mcd-transformers/transformers/shared"
 	"github.com/makerdao/vulcanizedb/pkg/core"
+	"github.com/makerdao/vulcanizedb/pkg/datastore/postgres"
 	. "github.com/onsi/ginkgo"
 	. "github.com/onsi/gomega"
 
```

Comment:
if you remove this space and run `goimports -w transformers/events/vat_flux/converter_test.go` it'll rearrange the deps better